src/client/Microsoft.Identity.Client/Instance/Discovery/RegionDiscoveryProvider.cs

```diff
@@ -56,9 +57,14 @@ private static InstanceDiscoveryMetadataEntry CreateEntry(string originalEnv, st
 
         private static string GetRegionalizedEnvironment(Uri authority, string region, RequestContext requestContext)
         {
-
             string host = authority.Host;
 
+            if (requestContext.ServiceBundle.Config.UseMtlsPop)
```

Comment:
What about the case of non-regional + mtls POP config? Do we want to support this or just error out?